Mathlib/LinearAlgebra/CliffordAlgebra/Grading.lean

```diff
@@ -93,21 +92,22 @@ theorem GradedAlgebra.lift_ι_eq (i' : ZMod 2) (x' : evenOdd Q i') :
     | add x y i hx hy ihx ihy =>
       -- Note: in https://github.com/leanprover-community/mathlib4/pull/8386 `map_add` had to be specialized to avoid a timeout
       -- (the definition was already very slow)
-      rw [AlgHom.map_add, ihx, ihy, ← AddMonoidHom.map_add]
+      rw [map_add, ihx, ihy, ← AddMonoidHom.map_add]
```

Comment:
I am pretty sure `AlgHom.map_add` was used for performance reasons...

Reply:
Looks like I correctly remembered

Reply:
The difference here is 5000 heartbeats, again inconsequential.

Reply:
Then you should remove the comment above